tests/components/dynalite/test_config_flow.py

```diff
@@ -91,5 +92,6 @@
         await hass.async_block_till_done()
     assert result["type"] == "abort"
     assert result["reason"] == "already_configured"
-    bridge = get_bridge_from_hass(hass)
+    entry_id = await get_entry_id_from_hass(hass)
+    bridge = hass.data[dynalite.DOMAIN][entry_id]
     assert bridge.dynalite_devices.port == port2
```

Comment:
We don't want to assert this on the bridge. It would be ok to patch the device library and assert that the library mock is called with this port.

Reply:
done